### ISSUE TITLE
I can't set expirations for caching

### ISSUE THREAD

**Issue by Jamesmjkim** [oslabs-beta/DenoStore#21]
Hello DenoStore team,

I was implementing your tool to my application and found that I cannot set expirations for the queries. I would like to put expiration times for some of my queries. 

Is there a solution your team is looking into?

**Comment by sktatsuno** [comment]
Thank you for the feedback, we are currently adding expiration specification to denostore cache method.

**Comment by sktatsuno** [comment]
You can now specify expiration time add the 'ex' property to the options object with a value of seconds until expire.

`denostore.cache({ info: info, ex: 5 ), callback)`

This will pass the expire time in seconds as an option to the redis set method and the cache will expire as instructed.

You can also now set a default expire time in seconds when setting up denostore. This default expire time will be used if no expire argument is specified when using the cache function in your GraphQL schema. If no expire argument or default expire is specified, the results will be cached with no expiration.

```
const denostore = new Denostore({
route: '/graphql',
usePlayground: true,
schema,
redisClient,
defaultEx: 10,
});
```

**Comment by sktatsuno** [comment]
User can now specify ex of -1 for queries they want no expiration time for:
`context.denostore.cache({ info: info, ex: -1 }, callback)`

**Comment by sktatsuno** [comment]
Closing issue as it is now addressed in #26
